go-controller/pkg/node/healthcheck.go

```diff
@@ -113,6 +113,10 @@ func checkForStaleOVSInterfaces(stopChan chan struct{}) {
 						val, stderr, err)
 				}
 			}
+		case <-time.After(10 * time.Second):
+			klog.Warningf("NUMS :   FOR TESTING ONLY ... TRIGGERING RECOMPUTE...")
```

Comment:
goanalysis_metalinter: 39.386011567s, unused: 4.357758791s"
##[error]pkg/node/healthcheck.go:118:32: Error return value of `util.RunOVNAppctlWithTimeout` is not checked (errcheck)
			util.RunOVNAppctlWithTimeout(5, "-t", "ovn-controller", "recompute")
			
The verify test is failing because if this